### DISCUSSION TITLE
Copilot chat stuck on collecting context in Jetbrains

### DISCUSSION THREAD

**Discussion by samuelgfeller** [community/community#129724] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Since a few weeks this keeps happening and it's very annoying. 
I thought for sure it would get fixed in an update but now multiple updates later the issue keeps on happening. 

v1.5.11.5872 after hibernation or wifi change I'm not sure but after getting back to work after some absence this happens all the time.
![image](https://github.com/community/community/assets/31797204/bed3b2ce-980a-4193-81bb-8a1ad84fea48)
restarting the IDE fixes the issue temporarily.

**Comment by samuelgfeller** [comment]
Happened 3 more times in the meantime with v1.5.11.5872

**Reply by tpope** [reply, marked as answer]
Please try 1.5.12.5909 and let us know if it still happens. It might finally be fixed. :crossed_fingers:

**Reply by samuelgfeller** [reply]
It seems to be good
